logic/broadcaster.js

```diff
@@ -102,6 +102,9 @@ Broadcaster.prototype.bind = function (peers, transport, transactions) {
 Broadcaster.prototype.getPeers = function (params, cb) {
 	params.limit = params.limit || self.config.peerLimit;
 	params.broadhash = params.broadhash || null;
+	params.matchBroadhash = params.matchBroadhash || false;
```

Comment:
Would be nice to describe those new params in function description.

Reply:
done